modules/oke/scripts/secret.py

```diff
@@ -38,8 +39,19 @@ def read_secret_value(secret_client, secret_id):
     delsecret = "kubectl -n default delete secret ${secret_name}"
     os.system(delsecret)
 
-    crtsecret = ("kubectl create secret docker-registry ${secret_name} -n default --docker-server=${region_registry} --docker-username=${tenancy_namespace}/${username} --docker-email=${email_address} --docker-password=%s" % secret_content)
-
+    create_namespace = f"""
```

Comment:
if namespace is not given it will try to create default . please add a validation to check namespace exists